docs/crio.conf.d.5.md

```diff
@@ -0,0 +1,25 @@
+% crio.conf.d(5)
+
+# NAME
+crio.conf.d - directory for drop-in configuration files
```

Comment:
nit: suggest rewrite:
`directory for drop-in configuration files for CRI-O`

Reply:
Done